README.md

```diff
@@ -1 +1,143 @@
+# hrxp-api
 
+API and backend for the HRX Project.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Database Tables](#database-tables)
+  - [Schema](#schema)
+- [Endpoints](#endpoints)
+- [Data requests and responses](#Data-requests-and-responses)
+
+# Getting Started
+
+- Run `npm i` from the command line.
+- Run `npm start`.
+
+# Database Tables
+
+## Schema
+
+[Back to table of Contents](#table-of-contents)
+
+# Endpoints
+
+### Channels
+
+| Endpoint            | METHOD | Description | Completed |
+| ------------------- | ------ | ----------- | :-------: |
+| [`/hello`](#/hello) | GET    |             |     X     |
+| []()                | GET    |             |           |
+
+### Users
+
+| Endpoint             | METHOD | Description | Completed |
+| -------------------- | ------ | ----------- | :-------: |
+| [`/users` ](#/users) | GET    |             |           |
+| []()                 | GET    |             |           |
+
+### messages
+
+| Endpoint                   | METHOD | Description | Completed |
+| -------------------------- | ------ | ----------- | :-------: |
+| [`/messages` ](#/messages) | GET    |             |           |
+| []()                       | GET    |             |           |
+
+# Data requests and responses
+
+Below are all expected request data responses.
+
+## `/channels`
+
+**Method:** GET
+
+Structure of request:
+
+```
+{
+
+}
+```
+
+**HTTP Status:**
```

Comment:
I think the `HTTP Status` can be group together, unless you're planning on having different fields for each request.

Reply:
Kyle is going to take care of the api documentation. But I see what you are saying